Formula/g/gitmux.rb

```diff
@@ -0,0 +1,25 @@
+class Gitmux < Formula
+  desc "Git status in tmux status bar"
+  homepage "https://github.com/arl/gitmux"
+  url "https://github.com/arl/gitmux/archive/refs/tags/v0.11.2.tar.gz"
+  sha256 "c62b180415c272743d01531b911091b9c35911be4ec4aae3e7bfceddf5094f6c"
+  license "MIT"
+
+  depends_on "go" => :build
+  depends_on "tmux"
```

Comment:
```suggestion
  depends_on "git" => :test
```

Reply:
@daeho-ro git is required for gitmux to work, not just for the test.